extensions/reactive-messaging-http/deployment/src/test/java/io/quarkus/reactivemessaging/http/sink/app/Dto.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.reactivemessaging.http.sink.app;
+
+public class Dto {
```

Comment:
Dto? Not a Person, Pet, or Pokemon object? 
You're making tests boring :-D

Reply:
I'll do better next time ;p